index.html

```diff
@@ -46,6 +46,207 @@ <h1>I'm Naan.
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text underline">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <button class="buttons">
+                                <div class="see_project">See project</div>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card_works">
+                    <div class="snapshot_portfolio">
+                    </div>
+                    <div class="left_block underline">
+                        <div class ="primary_text">
+                            <div class="project_title">Tonic</div>
+                            <div class="sub_header">
+                                <div class="client">
+                                    <div class="client_text">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <button class="buttons">
+                                <div class="see_project">See project</div>
+                            </button>
```

Comment:
- [ Required ] It's not semantically correct for an `inline` element in this case a `button` to be the parent of a block element in this case a `div`. Please consider making all the `See Project` buttons, just normal buttons as had been requested by the previous reviewer.

```HTML
  <button class="buttons">See Project</button>
```